### PR TITLE
[3829][UPD] dotfiles

### PR DESCRIPTION
**Pull request by liuhehe1995**
[3829](https://www.quartile.co/web#id=3829&menu_id=505&cids=3&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+              if [ -f ${reqfile} ] ; then
+                  result=0
+                  # reject non-comment lines that contain a / (i.e. URLs, relative paths)
+                  grep "^[^#].*/" ${reqfile} || result=$?
+                  if [ $result -eq 0 ] ; then
+                      echo "Unreleased dependencies found in ${reqfile}."
+                      exit 1
+                  fi
+              fi
+          done
```
Comment (AungKoKoLin1997) [inline .github/workflows/test.yml]: Remove this test as a comment.

**Review by AungKoKoLin1997 — COMMENTED** [review]
@liuhehe1995 Create test-requirements.txt and add odoo_test_helper like https://github.com/OCA/sale-workflow/blob/16.0/test-requirements.txt.

**Comment by AungKoKoLin1997** [comment]
The changes added by this PR https://github.com/qrtl/fal-oca/pull/25/files for adding `need_validation` in `if` condition is not valid and lead to ignoring check because `need_validation` will always be `false` when `review_ids` have value as per compute function of `need_validation` https://github.com/qrtl/fal-oca/blob/477a7a2decdca8f41c759836ca0358ea1ebadcbf/base_tier_validation/models/tier_validation.py#L195-L197.
So, this condition(https://github.com/qrtl/fal-oca/blob/477a7a2decdca8f41c759836ca0358ea1ebadcbf/base_tier_validation/models/tier_validation.py#L241-L249) will never be met and leading to OCB test failed.

**Review by AungKoKoLin1997 — APPROVED** [review]

**Comment by AungKoKoLin1997** [comment]
@rinaldifirdaus  Please help me to check the behavior you want to do is not changed by this PR.

**Review by yostashiro — COMMENTED** [review]
@AungKoKoLin1997 Can you please exclude `.void` directory from pre-commit (please make it our standard practice).

**Review by yostashiro — APPROVED** [review]

**Review by yostashiro — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]

**Pull request merged**